### PR TITLE
No issue use fixed skipper version

### PR DESCRIPTION
**Pull request by rccrdpccl**
Latest skipper has breaking changes that might break some jobs, this PR avoids that from happening

### CODE REVIEW

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml [hunk 1]
```diff
@@ -222,8 +222,6 @@ tests:
       ASSISTED_CONFIG: |
         SERVICE_BASE_REF=cloud_hotfix_releases
         STATIC_IPS=true
-        TEST_FUNC=test_networking
-      PACKET_PLAN: m3.large.x86
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml:226]: what's this change do?

Comment (rccrdpccl) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml:226]: ohhh I haven't even realized :facepalm: 
Leftovers... sorry

**Comment by rccrdpccl** [comment]
superseeded by https://github.com/openshift/release/pull/48510

**Pull request closed without merging**